### ISSUE TITLE
FIX: app deletes content on failed json entry

### ISSUE THREAD

**Issue by CJ-Wright** [regro/regolith#50]
If the user tries to put in bad json, then when the page is re-loaded then all their work is gone. This is rather nasty as they could have put a lot of work into their entry. Maybe we can reload the existing string? This may also be helped by moving to yml

**Comment by scopatz** [comment]
Yeah this sucks. I don't think yaml will help since by then it is loaded into memory

**Comment by CJ-Wright** [comment]
Could we have a stage in the app which looks for any files in the *sacred temp sub-folder* and then tries to read them in after the reading in of the database info?
